packages/dx-react-grid/docs/guides/column-resizing.md

```diff
@@ -27,3 +27,9 @@ In [uncontrolled mode](controlled-and-uncontrolled-modes.md), use the `TableColu
 In [controlled mode](controlled-and-uncontrolled-modes.md), pass the column widths to the `TableColumnResizing` plugin's `columnWidths` property, and handle the `onColumnWidthsChange` event to control the column width state.
 
 .embedded-demo({ "path": "grid-column-resizing/controlled", "showThemeSelector": true })
+
+### Resizing Mode
```

Comment:
I think we should extract this part from the `Basic` guide part. (`### Resizing Mode` -> `## Resize Modes``)